numpy/core/src/umath/ufunc_object.c

```diff
@@ -4555,42 +4631,15 @@ ufunc_generic_call(PyUFuncObject *ufunc, PyObject *args, PyObject *kwds)
     /* wrap outputs */
     for (i = 0; i < ufunc->nout; i++) {
         int j = ufunc->nin+i;
-        PyObject *wrap = wraparr[i];
+        _ufunc_context context = {ufunc, full_args, i};
```

Comment:
Appveyor complains about this line (https://ci.appveyor.com/project/charris/numpy/build/1.0.10809/job/3m1asbd6b1v588ro#L1894):
numpy\core\src\umath\ufunc_object.c(4634) : error C2440: 'initializing' : cannot convert from 'ufunc_full_args' to 'PyObject *'

Does look like the definition is indeed off.

Reply:
Looks like a bug in the old msvc needed by python 2.7 - guess I'll revert